src/more-infos/more-info-climate.html

```diff
@@ -46,22 +47,51 @@
         width: 100%;
       }
 
+      paper-range-slider {
+        width: 198px;
+        height: 2px;
+        margin-bottom: 15px;
```

Comment:
Material design margins go in increments of 8. So 16px should be appropriate here.